### ISSUE TITLE
Brightness contrast of the control boundary line is less than 3:1

### ISSUE THREAD

**Issue by Jessie-Zhang01** [dotnet/winforms#3322]
* .NET Core Version: .Net Core 5.0.100-preview.6.20270.51

* Have you experienced this same bug with .NET Framework?: 
Yes 

**Related test scenarios:**

- The contrast of control default status(non-focused, unselected).

  - Affected controls:
    - [x] Button
    - [x] ComboBox(DropDownStyle = DropDownList)
    - [x] TooStrip(ProgressBar, Button) 
    - [x] DataGridView(Inner boundary line)
    - [x] PropertyGrid
    - [x] RichTextBox
    - [x] ProgressBar
    - [x] DomainUpDown
    - [x] GroupBox
    - [x] NumericUpDown
    - [x] MonthCanlendar
    - [x] StatusStrip
    - [x] TabControl

- The contrast of control focused, unselected status.

  - Affected controls: 
    - [x] ContextMenuStrip
    - [x] MenuStrip
    - [x] ToolStrip (Button)

- The contrast of control focused, selected status.

  - Affected controls: 
    - [x] CheckedListBox
    - [x] ComboBox(DropDownStyle = DropDownList)
    - [x] ListBox
    - [x] ListView
    - [x] TreeView

**Minimal repro:**
1.	Install the Accessibility Insights from https://accessibilityinsights.io/en/downloads.
2.	Create a winforms core project.
3.	Add a control in the Form designer, build and run this project.
4.	Open the Accessibility Insights and select the Contrast of the directory on the left.
5.	Click on the color picker under Color1 and focus it to the border lines of the control.
6.	Click on the color picker under Color2 and focus it to the background of the control.

**Expected behavior:**
The brightness contrast of the control boundary line should be greater than 3:1.

**Actual behavior:**
Brightness contrast of the control boundary line is less than 3:1.
![image](https://user-images.githubusercontent.com/45864649/82546135-3da16180-9b8a-11ea-9933-4b57706ddd06.png)

**Comment by wjk** [comment]
I don't think we can do much to change this. These graphics are supplied by `uxtheme.dll`, which is a part of Windows. The underlying system cannot be changed or extended by anyone outside the Windows team. (That being said, if a High Contrast theme is being used, the bitmaps will change to follow the accessibility tenets of that mode, so there is that.)

**Comment by merriemcgaw** [comment]
Removing accessibility tags as conversation with @guybark indicates that our controls don't need to meet that boundary requirement because they are rather easy to figure out on their own.

**Comment by merriemcgaw** [comment]
I didn't know if we wanted to keep this open to evaluate as we look at the theming work we're considering.
